databricks/koalas/tests/test_dataframe.py

```diff
@@ -371,10 +371,26 @@ def test_missing(self):
                                         "Series.*{}.*not implemented".format(name)):
                 getattr(d.a, name)()
 
+    def test_to_numpy(self):
+        df = pd.DataFrame({'a': [4, 2, 3, 4, 8, 6],
+                           'b': [1, 2, 9, 4, 2, 4],
+                           'c': ["one", "three", "six", "seven", "one", "5"]},
+                          index=[10, 20, 30, 40, 50, 60])
+
+        ddf = koalas.from_pandas(df)
+
+        np.testing.assert_equal(ddf.to_numpy(), df.values)
```

Comment:
Since Pandas=23.4 doesn't have `to_numpy()` method. I used `values` property instead to pass the test.